### PR TITLE
Admin: fix admin URLs for Akismet, VaultPress and Scan foldable cards. Make strings available for translation.

### LINKED ISSUE #3990: New UI settings links not taking me to the right place.
In the new UI Some of the links that me to the wrong place. 

#### What I expected
When I clicked on VaultPress Settings it should take me to the VaultPress settings page. 
Also the UI should know if I have VaultPress installed already and if that is not the case include not show the links but maybe links to install the plugin. ?

screen shot 2016-05-26 at 13 37 33

#### What happened instead
Instead of being taken to https://example.com/aaa/wp-admin/admin.php?page=vaultpress
I get taken to https://example.com/wp-admin/admin.php?page=vaultpress Which is not right.

#### Steps to reproduce the issue
In current master. Go to the UI and click on the settings links.

### PR DESCRIPTION
**Pull request by eliorivero**
Fixes #3990

#### Adds correct URLs for Akismet and VaultPress settings in admin
- uses `Initial_State.adminUrl` it to build a proper URL to the admin page for Akismet and VaultPress

#### Check if Akismet/VaultPress plugins are activated and/or installed and display relevant messages.
- Introduces functions `isPluginActive` and `isPluginInstalled`
- New endpoint `/plugins` returns information about the plugins. Requires `activate_plugins` capability.
- New endpoint `/plugin/%plugin-slug%` returns information about the plugin. Requires `activate_plugins` capability.

messages

#### Translation
- makes some missing strings available for translation in site health

### CODE REVIEW

File: _inc/lib/admin-pages/class.jetpack-react-page.php [hunk 1]
```diff
@@ -123,6 +123,7 @@ function page_admin_scripts() {
 
 		// Add objects to be passed to the initial state of the app
 		wp_localize_script( 'react-plugin', 'Initial_State', array(
+			'WP_Admin' => admin_url(),
```
Comment (dereksmart) [inline _inc/lib/admin-pages/class.jetpack-react-page.php]: We already have this here.  Look below, it's sent over as `adminUrl`

Comment (eliorivero) [inline _inc/lib/admin-pages/class.jetpack-react-page.php]: Updated, thanks.

**Comment by dereksmart** [comment]
Looks good, but missing security scan?

**Comment by eliorivero** [comment]
@dereksmart the Scan card texts are now fixed in 954ddcc352c52fce11fa6ca0050b07875b8139eb

File: _inc/client/site-health/index.jsx [hunk 2]
```diff
@@ -17,12 +18,25 @@ import {
...
+import QuerySitePlugins from 'components/data/query-site-plugins';
 
-export const Page = ( { toggleModule, isModuleActivated, isTogglingModule, getModule } ) => {
+export const Page = ( props ) => {
+	let {
+			toggleModule,
+			isModuleActivated,
+			isTogglingModule,
+			getModule
+			} = props;
```
Comment (oskosk) [inline _inc/client/site-health/index.jsx]: There's an extra indent level here

**Comment by oskosk** [comment]
LGTM other than an indentation comment I left on the code

I'd say to merge it and if you will, add an issue for "Adding tests for the plugins state reducer" and (I can tackle the task, so if you wish assign it to me)

File: _inc/client/site-health/index.jsx (hunk 2 above)
Comment (eliorivero) [inline _inc/client/site-health/index.jsx]: Fixed indentation.

**Comment by oskosk** [comment]
:shipit:

**Pull request merged**